src/CP/valueselection/learning/environment.jl

```diff
@@ -41,3 +43,38 @@ function (learner::DQNLearner)(env::AbstractCPEnv{TS}) where {TS <: NonTabularTr
         vec |>
         send_to_host
 end
+
+function (learner::A2CLearner)(env::CPEnv{ST}) where {ST <: NonTabularTrajectoryState}
+    env |>
+    state |>
+    x ->
+        send_to_device(device(learner), x) |>
+        learner.approximator.actor |>
+        vec |>
+        send_to_host
+end
```

Comment:
We needed to redefine the forward pass, to work with `NonTabularTrajectoryState`